Classes/Service/RegistrationProcessor.php

```diff
@@ -169,6 +169,10 @@ public function createAdditionalPersons(Registration $registration, int $storage
             $person->setName($name);
             $email = (string)($singleUserData['email'] ?? '');
             $person->setEmail($email);
+            // Consider additional attendees as having just logged in, so set their last login to now
+            if (\method_exists($person, 'setLastLogin')) {
```

Comment:
This method always exists. (We might need to raise the feuserextrafields dependency for this - haven‘t checked.)